index.js

```diff
@@ -1,5 +1,6 @@
 const { eventLoopUtilization } = require('node:perf_hooks').performance;
 const { monitorEventLoopDelay } = require('node:perf_hooks');
+const debug = require('debug')('jambonz:mw-registrar');
```

Comment:
I think it should be drachtio:under-pressure

Reply:
fixed